setup.cfg

```diff
@@ -1,4 +1,21 @@
+[metadata]
+name = widgetastic.patternfly
+author = Milan Falesnik
+author_email = mfalesni@redhat.com
+description= Patternfly widget library for Widgetastic
+long_description = file: README.rst
```

Comment:
It will be great if we add content type as rest. It will help to render doc properly.
```
long_description_content_type='text/x-rst'
```